core/templates/dev/head/collection_editor/collection_node_editor_directive.html

```diff
@@ -1,35 +1,27 @@
 <script type="text/ng-template" id="inline/collection_node_editor_directive">
-  <div>
-    [<[getCollectionNode().getExplorationId()]>] <em><[getCollectionNode().getExplorationTitle()]></em>
+  <div class="collection-editor-node-title">
+    <[getCollectionNode().getExplorationTitle()]>
+    <span ng-click="deleteCollectionNode()" class="glyphicon glyphicon-remove-circle" aria-hidden="true">
```

Comment:
Is the ``aria-hidden`` right here? Shouldn't screenreaders be able to indicate to users that collection nodes can be deleted?